elide-core/src/main/java/com/yahoo/elide/core/dictionary/EntityDictionary.java

```diff
@@ -1095,6 +1095,28 @@ public void bindEntity(EntityBinding entityBinding) {
         return fieldOrMethod.getAnnotation(annotationClass);
     }
 
+    /**
+     * Return a list of field name for given annotations.
+     *
+     * @param entityClass     the entity class
+     * @param annotationClassList given annotation types
+     * @return Set of field names having given annotation.
+     */
+    public Set<String> getFieldNamesWithAnnotations(Type<?> entityClass,
```

Comment:
Any time we add a method to the dictionary, we should add a corresponding test in EntityDictionaryTest.

Reply:
👍